controllers/machinedeletionremediation_controller_test.go

```diff
@@ -146,6 +146,7 @@ var _ = Describe("Machine Deletion Remediation CR", func() {
 						{commonconditions.ProcessingType, metav1.ConditionFalse, remediationSkippedNodeNotFound},
 						{commonconditions.SucceededType, metav1.ConditionFalse, remediationSkippedNodeNotFound}})
 					verifyConditionUnset(commonconditions.PermanentNodeDeletionExpectedType)
+					verifyEvent("Warning", "RemediationSkippedNodeNotFound", "failed to fetch node")
```

Comment:
Instead of `"Warning"` we can use `corev1.EventTypeWarning`, and `corev1.EventTypeNormal` instead of "Normal"